lib/widgets/pages/tickets_page.dart

```diff
@@ -0,0 +1,11 @@
+import 'package:coffeecard/base/style/text_styles.dart';
+import 'package:flutter/material.dart';
+
+class TicketsPage extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    return Center(
+      child: Text('Buy and spend tickets here!', style: AppTextStyle.sectionTitle),
```

Comment:
I know this is properly just a test string but we can just as well start up some good practices. According to [flutter documentation](https://flutter.dev/docs/get-started/flutter-for/android-devs#where-do-i-store-strings-how-do-i-handle-localization) as `static final` or `const` fields in its own class.

I suggest we create a new class `StringsConstants` or something similar under a new subpackage in the `base` package.

Reply:
Added Strings in lib/base.